src/intermediate.rs

```diff
@@ -186,9 +186,16 @@ impl<'a> IntermediateTypes<'a> {
                     unimplemented!("what to do here?");
                 }
             },
-            RustStructType::Wrapper { min_max: Some(_) , ..} => {
+            RustStructType::Wrapper { control: Some(ControlOperator::Range(_)) , ..} => {
                 self.mark_new_can_fail(rust_struct.ident.clone());
             },
+            RustStructType::Wrapper { control: Some(ControlOperator::Control(_)) , ..} => {
+                self.mark_new_can_fail(rust_struct.ident.clone());
+                // match ctl {
```

Comment:
Wasn't sure how to handle this case (do I need `mark_new_can_fail` at all? It crashes if I don't add it though)